src/pages/LoginPage.js

```diff
@@ -4,15 +4,15 @@ import LoginCard from '../components/forms/login/LoginCard'
 import Alert from '../components/alert/Alert'
 
 const LoginPage = () => {
-  const [alert, setAlert] = useState({ show: false, message: '', type: '' })
-  const handleAlert = (value)=>{
-    setAlert(value)
-  }
+  // const [alert, setAlert] = useState({ show: false, message: '', type: '' })
+  // const handleAlert = (value)=>{
+  //   setAlert(value)
+  // }
   return (
     <>
-    {alert.show && <Alert type={alert.type} message={alert.message} />}
+    {/* {alert.show && <Alert type={alert.type} message={alert.message} />} */}
```

Comment:
i think the approach of showing the alert on the page file and not on the card is better..
this can be even be done such that, the alert is part of the auth layout or dependent on it..
so it shows at a fixed places